decision-record/DR009-who-can-access-authentication

```diff
@@ -0,0 +1,21 @@
+# DR009: Deciding who can access GOV.UK Forms
+
+Date: 2023-06-27
+
+## Status
+
+> Proposed
+
+## Context
+
+> While looking at replacing Signon with either CDDO's SSO, or Auth0, it was raised that restricting users to gov.uk email accounts only would still allow non-central government deparmtnets, such as local councils, access to create an account.
```

Comment:
```suggestion
> While looking at replacing Signon with either CDDO's SSO, or Auth0, it was raised that restricting users to gov.uk email accounts only would still allow non-central government departments, such as local councils, access to create an account.
```

Reply:
Whoops, thanks for spotting! Might change that to 'organisations' anyway as a local council isn't a department.